vars/FreeBSD.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+default_pdns_package_name: "powerdns"
+
+default_pdns_config_dir: "/usr/local/etc/pdns"
+
+pdns_user: root
+pdns_group: wheel
+
+pdns_fileowner:
+  user: root
+  group: wheel
+
+
+# The name of the PowerDNS Authoritative Server debug package
+default_pdns_debug_symbols_package_name: "pdns-server-dbg"
+
+# List of PowerDNS Authoritative Server Backends packages on Debian
+default_pdns_backends_packages:
```

Comment:
why is this list for Debian in vars/FreeBSD.yml?

Reply:
(I understand it was copied from the Debian file, but it doesn't make sense for FreeBSD)

Reply:
The conditionals from tasks/install-FreeBSD.yml break is the variables are not defined. 
`
name: Install the PowerDNS Authoritative Server debug symbols
...
when: pdns_install_debug_symbols_package
`
`
name: Install the PowerDNS Authoritative Server backends
...
when: pdns_backends_packages is defined
`

That was the only way I found (within the time I put into it) to solve the issue. There is almost certainly a better way.

Reply:
I checked some more, and my other suggestion would be to change the conditionals to 
`when: not pdns_install_debug_symbols_package_name` and set the default to be false. 
When a package name is entered, it logs that the feature isnt supported. That was along the lines I had in mind.